source/game/g_main.cpp

```diff
@@ -701,7 +701,7 @@ static cvarTable_t		gameCvarTable[] = {
 	{ &g_noSpecMove, "g_noSpecMove", "0", CVAR_SERVERINFO, 0, qtrue },
 
 	// noset vars
-	{ NULL, "gamename", GAMEVERSION , CVAR_SERVERINFO | CVAR_ROM, 0, qfalse  },
+	{ NULL, "gamename", GAMEVERSION , CVAR_SERVERINFO | CVAR_ROM, qfalse, qfalse  },
```

Comment:
This sucker right here may or may not be related to another issue I came across, just commenting here so I can find it again down the line if I have to.